### PR TITLE
Upgrade phpunit

### PR DESCRIPTION
**Pull request by jdrieghe**
### Changed
- Upgraded phpunit to ^7.5

### Fixed
- Incorrect/missing test namespace configuration

### CODE REVIEW

File: tests/CalendarHTMLFormatterTest.php [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
-    public function testGeneralFormatMethod()
+    public function testGeneralFormatMethod(): void
     {
         $offer = new Event();
         $offer->setCalendarType(Event::CALENDAR_TYPE_SINGLE);
         $offer->setStartDate(new \DateTime('2018-01-25T20:00:00+01:00'));
         $offer->setEndDate(new \DateTime('2018-01-25T21:30:00+01:00'));
 
-        $this->formatter->format($offer, 'xs');
+        $this->assertIsString($this->formatter->format($offer, 'xs'));
```
Comment (lucwollants) [inline tests/CalendarHTMLFormatterTest.php]: Is it worth also checking the actual string versus the expected string?

Comment (lucwollants) [reply tests/CalendarHTMLFormatterTest.php]: This question applies to other `assertIsString` occurrences.

**Comment by lucwollants** [comment]
I still noticed a few constructors of `ExtraSmallMultipleHTMLFormatter` with the second parameter, which is now removed. For example inside:
- tests/Multiple/ExtraSmallMultipleHTMLFormatterTest.php
- tests/Multiple/ExtraSmallMultiplePlainTextFormatterTest.php
- tests/Multiple/SmallMultipleHTMLFormatterTest.php
- tests/Multiple/SmallMultiplePlainTextFormatterTest.php

File: tests/CalendarHTMLFormatterTest.php (hunk 1 above)
Comment (jdrieghe) [reply tests/CalendarHTMLFormatterTest.php]: Not really necessary because they were already covered by other tests, but I added it now

**Comment by lucwollants** [comment]
I also noticed that the tests inside the `Periodic` subfolder are missing `Tests` in their namespace. This also applies to the `Permanent` folder.

File: tests/Permanent/LargePermanentPlainTextFormatterTest.php [hunk 2]
```diff
@@ -5,24 +5,25 @@
 use CultuurNet\CalendarSummaryV3\Translator;
```
Comment (lucwollants) [inline tests/Permanent/LargePermanentPlainTextFormatterTest.php]: This use statement is obsolete.

**Review by lucwollants — APPROVED** [review]
Clean!

**Pull request merged**